core/invoker/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -118,7 +118,7 @@ class Invoker(
                 }
 
                 info(this, s"activation failed")
-                completeTransactionWithError(actionid.id, msg.action.version, tran, failure.activationResponse)
+                completeTransactionWithError(actionid.id, msg.action.version.get, tran, failure.activationResponse)
```

Comment:
Let's make sure this is defined - add a require version is defined at entry to onMessage.
Squash related changes to into this commit (if possible).